pipelines/run-pipelines-on-apache-airflow/README.md

```diff
@@ -0,0 +1,326 @@
+
+1. Open the JupyterLab Launcher.
+1. Click the `Apache Airflow pipeline editor` tile to open the Visual Pipeline Editor for Apache Airflow.
+1. Expand the palette panel. Two new component categories are displayed (`request` and `scripting`), each containing one component entry that you added:
+
+   ![Palette with custom components](doc/images/palette-with-custom-components.png)
+   
+1. Drag the '`SimpleHttpOperator`' component onto the canvas to create the first pipeline node.
+1. Drag the '`BashOperator`' component onto the canvas to create a second node and connect the two nodes as shown.
+
+   ![Pipeline with two custom components](doc/images/wip-tutorial-pipeline.png)
+
+   The components require inputs, which you need to specify to render the nodes functional.
+1. Open the properties of the '`SimpleHttpOperator`' node:
+      - select the node and expand (&#8612;) the properties slideout panel on the right OR
+      - right click on the node and select `Open Properties`
+    
+   ![](doc/images/review-http-node-properties.png)
+
+
+1. Review the node properties. The properties are a combination of Elyra metadata and information that was extracted from the [component's specification](https://raw.githubusercontent.com/elyra-ai/examples/master/pipelines/run-pipelines-on-apache-airflow/components/http_operator.py):
+   ```
+   class SimpleHttpOperator(BaseOperator):
+    """
+    Calls an endpoint on an HTTP system to execute an action
+
+    :param http_conn_id: The connection to run the operator against
+    :type http_conn_id: str
+    :param endpoint: The relative part of the full url. (templated)
+    :type endpoint: str
+    :param method: The HTTP method to use, default = "POST"
+    :type method: str
+    :param data: The data to pass. POST-data in POST/PUT and params
+        in the URL for a GET request. (templated)
+    :type data: For POST/PUT, depends on the content-type parameter,
+        for GET a dictionary of key/value string pairs
+    :param headers: The HTTP headers to be added to the GET request
+    :type headers: a dictionary of string key/value pairs
+    :param response_check: A check against the 'requests' response object.
+        Returns True for 'pass' and False otherwise.
+   ...
+   ```
+
+   The Elyra properties include:
+
+   - `Label`: If specified, the value is used as node name in the pipeline instead of the component name. Use labels to resolve naming conflicts that might arise if a pipeline uses the same component multiple times. For example, if a pipeline utilizes  the '`SimpleHttpOperator`' component to make two requests, you could override the node name by specifying '`HTTP Request 1`' and '`HTTP Request 2`' as labels:
+
+      ![Use labels to produce unique node names](doc/images/label-example.png)
+
+   - `Component source`: A read-only property that identifies source information from where the component specification was loaded. This property is displayed for informational purposes only.
```

Comment:
May want to add more information to this property since it is much more complex than it was in 3.2